packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -438,7 +438,8 @@ export class BasePickerListBelow<T, P extends IBasePickerProps<T>> extends BaseP
         { this.renderSuggestions() }
         <FocusZone ref={ this._resolveRef('focusZone') }
           className='ms-BasePicker-selectedItems'
-          isInnerZoneKeystroke={ this._isFocusZoneInnerKeystroke } >
+          isInnerZoneKeystroke={ this._isFocusZoneInnerKeystroke }
+          direction={ FocusZoneDirection.horizontal } >
```

Comment:
If that is the case I can back this portion out, however there are a couple issues that will need attention if we don't switch this to horizontal:

1. Even with Edge's alignment issues fixed, the underlying issue still makes some of the elements unfocusable.
2. In all browsers, keyboarding is completely broken in the example custom document picker on the Pickers component page. Only the first selected document is focusable and none of the internal elements can be selected.

Switching to horizontal remedies both of these problems, but at the expense of the up and down keys. So it may be a cleaner, more permanent solution to address these two issues separately instead of using horizontal here. My suspicion is that issue 1 is an Edge bug and issue 2 may be a problem with that specific picker implementation.